### ISSUE TITLE
Blinds are not working anymore since update

### ISSUE THREAD

**Issue by Arjan1993** [Xenomes/Domoticz-TUYA-Plugin#36]
I have some tuya wifi blinds motors in my house. They have worked for several months but I think since the update of 12 november none of them is working anymore. In the tuya app they do working but not in domoticz. I have updated the plugin and domoticz to the latest version but its doesnt help for me.

**Comment by Xenomes** [comment]
What is the version of Domoticz?

**Comment by Arjan1993** [comment]
I`m using version 2023.1

**Comment by Xenomes** [comment]
Update the script, please test it out i don't have covers/blinds to test.

**Comment by Arjan1993** [comment]
First of all, many thanks for the plugin and the help.

I have changed the script plugin.py to the new plugin.py from here.
After that tuya stopt working. I have restarted the raspberry and domoticz but tuya wont start.
When I open plugin.py and run the script I get the following syntax error: 

Traceback (most recent call last):
  File "/home/pi/domoticz/plugins/tuyaha/plugin.py", line 157
    elif Command == 'Set Color':
       ^

So Ive looked at line 157 and saw that line wasn`t in the correct possition. So Ive moved it backwards 
same as the other elif statemands. After that the error was gone but non of my tuya devices are responding.
There are no errors in the log so I dont know where to look for information.

Is it correct that when I do a git pull command the return is already up to date even before updating the script manualy?
